### PR TITLE
DOCSP-42020: queues feedback 5.0

### PR DESCRIPTION
**Pull request by rustagir**
https://jira.mongodb.org/browse/DOCSP-42020

Replaces `collection()` with `table()`
Adds sentence about using default failed jobs provider from Laravel

[Staging](https://deploy-preview-153--docs-laravel.netlify.app/queues/)

### Checklist

- [ ] Add tests and ensure they pass

### CODE REVIEW

File: docs/queues.txt [hunk 1]
```diff
@@ -11,6 +11,16 @@ Queues
 .. meta::
    :keywords: php framework, odm, code example, jobs
 
+Overview
+--------
+
+In this guide, you can learn how to use MongoDB as your database for
+Laravel Queue. Laravel Queue allows you to create queued jobs that can
+processed in the background.
```
Comment (jordan-smith721) [inline docs/queues.txt]:
```suggestion
Laravel Queue. Laravel Queue allows you to create queued jobs that are
processed in the background.
```

**Review by jordan-smith721 — APPROVED** [review]
LGTM +1 fix

**Review by GromNaN — APPROVED** [review]

File: docs/queues.txt [hunk 2]
```diff
@@ -69,7 +79,7 @@ collection:
    'failed' => [
        'driver' => 'mongodb',
        'database' => 'mongodb',
```
Comment (hans-thomas) [inline docs/queues.txt:81]: Replace `connection` with `database` in the table.

File: docs/queues.txt [hunk 3]
```diff
@@ -124,7 +133,7 @@ application's ``config/queue.php`` file:
     'batching' => [
        'driver' => 'mongodb',
        'database' => 'mongodb',
```
Comment (hans-thomas) [inline docs/queues.txt:135]: Also for job batching, replace `connection` with `database`.

**Review by norareidy — APPROVED** [review]

**Pull request merged**